incontext.py

```diff
@@ -50,11 +88,33 @@ def context_function(name=None):
     If `name` is not specified, the function name is used.
     """
     def decorator(f):
-        CONTEXT_FUNCTIONS[name if name is not None else f.__name__] = f
+        _PLUGINS.add_plugin(PLUGIN_TYPE_CONTEXT_FUNCTION, name if name is not None else f.__name__, f)
         return f
     return decorator
 
 
+def command(name, help=None, arguments=[]):
```

Comment:
This method should have some unit / integration tests.

Reply:
It's also important to ensure that both plugin discovery mechanisms work, especially when we've transitioned away from the old discovery mechanism and don't use it in the build-in plugins.

Reply:
See https://github.com/inseven/incontext/issues/63.